### PR TITLE
[refactor] test's location

### PR DESCRIPTION
**Pull request by kazukazuinaina**
I have updated a part of the BannedApiAnalyzer test code.
I think this code makes the part about Location more explicit.

### CODE REVIEW

**Review by sharwell — CHANGES_REQUESTED** [review]
First of all, I like the change! 👍 The new markup syntax is definitely better for test readability.

This issue appears to occur multiple times in the file (either that or the methods are now unused). Can you remove the following two methods and fix all remaining cases in this file?

https://github.com/dotnet/roslyn-analyzers/blob/065a674d0e102af7f2b78976b988157babcb0b4d/src/Microsoft.CodeAnalysis.BannedApiAnalyzers/UnitTests/SymbolIsBannedAnalyzerTests.cs#L28-L31

https://github.com/dotnet/roslyn-analyzers/blob/065a674d0e102af7f2b78976b988157babcb0b4d/src/Microsoft.CodeAnalysis.BannedApiAnalyzers/UnitTests/SymbolIsBannedAnalyzerTests.cs#L38-L41

**Comment by kazukazuinaina** [comment]
@sharwell 

Thanks for your review!
OK, I got it.

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/roslyn-analyzers?pullRequest=5348) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/roslyn-analyzers?pullRequest=5348) before we can accept your contribution.<br/><br/>:x: kazukazuinaina [sign now](https://cla.dotnetfoundation.org/dotnet/roslyn-analyzers?pullRequest=5348)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/roslyn-analyzers?pullRequest=5348) it.</sub>

File: src/Microsoft.CodeAnalysis.BannedApiAnalyzers/UnitTests/SymbolIsBannedAnalyzerTests.cs [hunk 1]
```diff
@@ -1348,12 +1338,12 @@ End Class
             await VerifyBasicAnalyzerAsync(
                 source,
                 bannedText1,
-                GetBasicResultAt(7, 13, 24, SymbolIsBannedAnalyzer.SymbolIsBannedRule, "Public Sub Banned()", ""));
+                GetBasicResultAt(0, ymbolIsBannedAnalyzer.SymbolIsBannedRule, "Public Sub Banned()", ""));
```
Comment (sharwell) [inline src/Microsoft.CodeAnalysis.BannedApiAnalyzers/UnitTests/SymbolIsBannedAnalyzerTests.cs:1341]:
```suggestion
                GetBasicResultAt(0, SymbolIsBannedAnalyzer.SymbolIsBannedRule, "Public Sub Banned()", ""));
```

**Review by sharwell — APPROVED** [review]

File: src/Microsoft.CodeAnalysis.BannedApiAnalyzers/UnitTests/SymbolIsBannedAnalyzerTests.cs (hunk 1 above)
Comment (kazukazuinaina) [reply src/Microsoft.CodeAnalysis.BannedApiAnalyzers/UnitTests/SymbolIsBannedAnalyzerTests.cs]: Thanks, I fixed.

**Pull request merged**